### DISCUSSION TITLE
Кодировка

### DISCUSSION THREAD

**Discussion by chiapch** [community/community#139666] [Programming Help]
### Body

Я пытаюсь отправить запрос с простой текстовой информацией на русском языке на свой локальный сервер, чтобы сервер просто принял ее хотя бы, но на русском языке возникают проблемы с кодировкой, на английском все работает, но как сделать на русском? отправляется js объект 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by abdessalems** [comment]
response 1

**Comment by abdessalems** [comment]
response 2

**Comment by ebndev** [comment]
Hi @chiapch, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
